torchao/quantization/quant_api.py

```diff
@@ -1900,6 +1957,30 @@ def _fpx_weight_only_transform(
     return module
 
 
+@dataclass
+class AOPerModuleConfig(AOBaseConfig):
+    module_name_to_config: Dict[str, Optional[AOBaseConfig]] = field(
+        default_factory=dict
+    )
+
+
+def _ao_per_module_config_handler(
```

Comment:
acutally what does this do?

Reply:
so for each module (+ module name), just checks the dictionary mapping from module name / fqn to specific configs, if there is a match, we apply the config in dictionary to the module.

Reply:
Ohh this makes total sense thanks